.github/workflows/test.yml

```diff
@@ -19,7 +19,8 @@ jobs:
         search-server-image:
           - docker.elastic.co/elasticsearch/elasticsearch-oss:7.10.0
           - opensearchproject/opensearch:1.1.0
-          - opensearchproject/opensearch:1.2.3
+          - opensearchproject/opensearch:1.2.4
+          - opensearchproject/opensearch:1.3.1
```

Comment:
Can we also add tests for all other versions of OpenSearch? 1.0.0, 1.0.1, 1.2.1, 1.2.2, 1.2.3, 1.3.1